CONTRIBUTING.md

```diff
@@ -0,0 +1,112 @@
+# Welcome to CONTRAST RATIO APP's contributing guide
+
+:+1::tada: First off, thank you for investing your time in contributing to our project! :tada::+1:
```

Comment:
Maybe it's an issue on my end, but
in VS code markdown previewer I see this:
![17 08 2022_15 20 37_REC](https://user-images.githubusercontent.com/108119145/185144912-5aa2d7b6-24c0-41af-b361-793ce515bc9a.png)

Test of 'tada'

:+1::tada: First off, thank you for investing your time in contributing to our project! :tada::+1:

Ok, on GitHub it looks fine. So I guess it's fine.

Reply:
Yeah, it doesn't show with the default VS code markdown preview.
You can install the following extension and it will show:
https://marketplace.visualstudio.com/items?itemName=shd101wyy.markdown-preview-enhanced